packages/geo/src/lib/geometry/geometry-form-field/geometry-form-field-input.component.ts

```diff
@@ -150,6 +209,27 @@ export class GeometryFormFieldInputComponent implements OnInit, OnDestroy, Contr
     return this.olOverlayLayer.getSource();
   }
 
+  @Input()
+  set radius(value: any) {
+    if (this.ready === false) {
+      return;
+    }
+    if (this.modifyControl.getSource()) {
+      this.modifyControl.getSource().refresh();
+    }
+    if (this.freehandDrawIsActive) {
+      let olModify;
+      setTimeout(() => {
+        olModify = this.modifyControl.olModifyInteraction;
```

Comment:
You could make a public method on the modify control (clearFeatures) instead of making the control public. Also, why is the setTimeout needed?

Reply:
It seems that the control interaction is not fill before this setter. Only there to assure the modifyInteraction is not undefined before clearing the features from it.